.github/workflows/ci-tools.yaml

```diff
@@ -10,14 +10,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # 16 is min version recommended to users
-        # 18 released 2022-04-19
-        # 19 released 2022-10-18
-        node-version: [ 19 ]
+        # even numbers are LTS; ods go to ~3m maintenance after next major version
+        # 16 is min version recommended to users; but unmaintained since 2023-10
+        # 18 released 2022-04-19 - maintained until June 2025
+        # 20 maintained until June 2026
+        # 21 is latest as of March 2023, with 22 coming in May
+        node-version: [ 18, 20, latest]
```

Comment:
👍 also, we should recommend using LTS versions somewhere in the docs... to avoid those Node.js [deprecation notices](https://app.asana.com/0/1206296312199462/1206355664470094/f)